data/io.github.lainsce.Notejot.appdata.xml.in

```diff
@@ -52,6 +53,14 @@
         <content_attribute id="money-purchasing">none</content_attribute>
         <content_attribute id="money-gambling">none</content_attribute>
     </content_rating>
+    <recommends>
+      <control>keyboard</control>
+      <control>pointing</control>
+      <control>touch</control>
+    </recommends>
+    <requires>
+      <display_length compare="ge">medium</display_length>
```

Comment:
This option means that at least laptops and tablets are required, as in the previous one, I don't have a touch screen to test on smaller devices so perhaps it would be better to leave it as a recommendation.